src/Storefront/Resources/app/storefront/src/scss/layout/_header.scss

```diff
@@ -9,28 +9,37 @@ The header contains the shop logo, the search form and various actions like the
     padding: $spacer-sm 0;
 }
 
-.header-logo-col {
-    margin: $spacer 0;
-}
+@if not feature('v6.7.0.0') {
+    .header-logo-col {
+        margin: $spacer 0;
+    }
 
-.header-logo-main {
-    text-align: center;
-    display: block;
-}
+    .header-logo-main {
+        text-align: center;
+        display: block;
+    }
+
+    .header-logo-main-link {
+        width: 100%;
```

Comment:
This has no effect at all, as it is on an inline-element (`anchor`). Hence I did not add it as bootstrap class.